crates/ty_python_semantic/resources/mdtest/type_properties/is_subtype_of.md

```diff
@@ -2207,6 +2207,50 @@ static_assert(is_subtype_of(CallableTypeOf[overload_ab], CallableTypeOf[overload
 static_assert(is_subtype_of(CallableTypeOf[overload_ba], CallableTypeOf[overload_ab]))
 ```
 
+### Generic callables
+
+A generic callable can be considered equivalent to an intersection of all of its possible
+specializations. That means that a generic callable is a subtype of any particular specialization.
+(If someone expects a function that works with a particular specialization, it's fine to hand them
+the generic callable.)
+
+```py
+from typing import Callable
+from ty_extensions import CallableTypeOf, TypeOf, is_subtype_of, static_assert
+
+def identity[T](t: T) -> T:
+    return t
+
+# TODO: no error
+# error: [static-assert-error]
```

Comment:
I assume that (to the extent the constraint-set versions of these tests pass correctly) we would expect these TODOs to be fixed by https://github.com/astral-sh/ruff/pull/20093 ?